docs/collecting-data/collecting-from-own-applications/python-tracker/tracking-specific-events/index.md

```diff
@@ -1,528 +1,282 @@
-| [`track_struct_event()`](#track-structured-events) | Track a Snowplow custom structured event |
-| [`track_self_describing_event()`](#track-self-describing-event) | Track an event that you have defined yourself |
-
-### Common tracking parameters
-
-All events are tracked with specific methods on the tracker instance, of the form `track_XXX()`, where `XXX` is the name of the event to track. The parameters that are common for all track methods are:
-
-- `context`
-- `tstamp`
-- `event_subject` (**_\*\*New to v0.9.0_**)
-
-#### 1. Custom context
-
-Custom context can be used to augment any standard Snowplow event type, including self describing events, with additional data.
-
-Custom context can be added as an extra argument to any of Snowplow’s `track..()` methods.
-
-When you track an event, some of the data that you track will be specific to that event. A lot of the data you want to record, however, will describe entities that are tracked across multiple events. For example, a media company might want to track the following events:
-
-- User views video listing
-- User plays video
-- User pauses video
-- User shares video
-- User favorites video
-- User reviews video
-
-Whilst each of those events is a different type, all of them involve capturing data about the user and the video. Both the 'user' and 'video' are entities that are tracked across multiple event types. Both are candidates to be "custom context". You as a Snowplow user can define your own custom contexts (including associated schemas) and then send data for as many custom contexts as you wish with _any_ Snowplow event. So if you want, you can define your own "user context", and then send additional user data in that object with any event. Other examples of context include:
-
-- articles
-- videos
-- products
-- categories
-- pages / page_types
-- environments
-
-Each tracking method accepts an additional optional context parameter after all the parameters specific to that method:
-
-The `context` argument should consist of an array of one or more instances of `SelfDescribingJson` class. This class isomorphic to self-describing JSON, to be more precisely - it has Iglu URI attribute and data itself.
-
-For example, if a server-side Python application can determine visitor's geoposition, this can be attached to the event, using the `geolocation_context` that is predefined on [Iglu Central](https://github.com/snowplow/iglu-central):
-
-```python
-from snowplow_tracker import SelfDescribingJson
+To track an event, simply pass the `Event` object to the `tracker.track()` method. For example, tracking a page view:
 
-geo_context = SelfDescribingJson(
-  "iglu:com.snowplowanalytics.snowplow/geolocation_context/jsonschema/1-0-0",
-  {
-    "latitude": -23.2,
-    "longitude": 43.0
-  }
+```python 
+page_view = PageView(
+  page_url="https://www.snowplow.io",
+  page_title="Homepage",
 )
+t.track(page_view) 
```

Comment:
`t` is not defined on this page, so worth spelling it out to `tracker` (better not to use any abbreviations in code especially in the docs snippets).

```suggestion
tracker.track(page_view) 
```